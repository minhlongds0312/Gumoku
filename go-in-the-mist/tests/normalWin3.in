place A1
place B10
place B2
place B12
place C3
place B11
place D4
place B13
place E5
