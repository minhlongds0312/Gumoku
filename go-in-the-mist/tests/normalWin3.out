Black wins!
A1B10B2B12C3B11D4B13E5
Thank you for playing!